python/kwiver/vital/algo/trampoline/estimate_pnp_trampoline.txx

```diff
@@ -29,10 +29,10 @@
  */
 
 /**
- * \file estimate_homography_trampoline.txx
+ * \file estimate_pnp_trampoline.txx
```

Comment:
This looks like an error in the documentation for PNP python bindings.  You could push just the changes to this file in a separate PR if you'd like.  That would be merged quickly.  Generally bug fixes like this should be pushed as separate PR's and rebased onto the `release` branch and then targeted for merging into the `release` branch.  This way they land sooner and are back ported to fix the last release.

Reply:
reverting this back to the erroneous line seems weird, but I can see the merge to release logic

Reply:
You don't need to revert this change here, although you could.  I'm just suggesting factoring it out so that it also stands alone an can be fast-tracked into merging in the release branch.  If you have this change on a separate commit you could just cherry-pick that commit onto a new branch off of the release branch and push that as a new PR.